dockers/docker-base-buster/etc/supervisor/supervisord.conf

```diff
@@ -3,8 +3,6 @@
 [unix_http_server]
 file=/var/run/supervisor.sock  ; (the path to the socket file)
 chmod=0700                     ; socket file mode (default 0700)
-username=dummy
-password=dummy
```

Comment:
this is removed because the dependent startup plugin will complaint about not authorized. since it is within the docker, I do not feel we need these dummy password anyway.